samples/cpp/tutorial_code/objectDetection/objectDetection.cpp

```diff
@@ -13,8 +13,8 @@ using namespace cv;
 void detectAndDisplay( Mat frame );
 
 /** Global variables */
-String face_cascade_name = "haarcascade_frontalface_alt.xml";
-String eyes_cascade_name = "haarcascade_eye_tree_eyeglasses.xml";
+String face_cascade_name = "../../data/haarcascades/haarcascade_frontalface_alt.xml";
+String eyes_cascade_name = "../../data/haarcascades/haarcascade_eye_tree_eyeglasses.xml";
```

Comment:
I agree with @sovrasov I find it very weird that there are still samples out there that have hard coded paths internally. It has more sense if people provide them using CmdLineParser